workspaces/orchestrator/plugins/orchestrator-form-react/src/utils/generateUiSchema.ts

```diff
@@ -110,9 +111,19 @@ function extractUiSchema(mixedSchema: JSONSchema7): UiSchema<JsonObject> {
     } else {
       processLeafSchema(curSchema, path);
     }
+    processOrder(curSchema, path);
     processComposedSchema(curSchema, path);
   };
 
+  const processOrder = (curSchema: JSONSchema7Definition, path: string) => {
```

Comment:
would be nice to have this in the generateUitSchema unit test, this will also explain this code

Reply:
but not a must